src/AspNet.Security.OAuth.Autodesk/AutodeskAuthenticationHelper.cs

```diff
@@ -0,0 +1,93 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using Newtonsoft.Json.Linq;
+
+namespace AspNet.Security.OAuth.Autodesk
+{
+    /// <summary>
+    /// Contains static methods that allow to extract user's information from a <see cref="JObject"/>
+    /// instance retrieved from Autodesk after a successful authentication process.
+    /// </summary>
+    public static class AutodeskAuthenticationHelper
+    {
+        /// <summary>
+        /// Key for claim containing a boolean value indicating whether the user's email has been verified
+        /// </summary>
+        public static readonly string EmailVerifiedClaim = "urn:autodesk:emailverified";
```

Comment:
Feel free to move these 2 constants to a new `AutodeskAuthenticationConstants` file, in a nested `Claims` class.